### DISCUSSION TITLE
Loading a Tycho project in 2022-09

### DISCUSSION THREAD

**Discussion by jeffmaury** [eclipse-m2e/m2e-core#929] [General]
My team has a lot of Tycho based project and we previously used m2eclipse-tycho when loading those Maven projects into Eclipse IDE. But as m2eclipse-tycho is not compatible anymore with M2E in 2022-09, what is the recommended way now ?
I tried on a newly created bundle and project was created as a Maven project but not as a Java or plugin project

**Comment by HannesWell** [comment]
If those projects are regular 'Plug-in projects' created/build with Eclipse PDE then the `m2eclipse-tycho` connector is not necessary at all and can be just uninstalled.
Furthermore I recommend to use at least Tycho 2.7.4, which has M2E lifecycle mappings embedded that silence the warnings about missing mappings.

**Reply by jeffmaury** [reply]
They are not yet eclipse projects just stock maven/tycho projects that I want to import

**Reply by HannesWell** [reply]
Sounds like your set up is similar to the one here? https://github.com/eclipse-m2e/m2e-core/issues/605#issuecomment-1214633577
Could you provide a minimal reproducer?
We could then think about a solution and maybe enhance the m2e.pde.connector accordingly. That connector is heavily based on the m2eclipse-tycho connector. If so please use the mentioned issue for that.

**Reply by jeffmaury** [reply]
The project that I mentioned is here: https://github.com/jbosstools/jbosstools-javaee/pull/494/files#diff-205cb3e540772e9ee9e9954a6995d879960b3b8de810fb0e4228d59741e59861

**Comment by HannesWell** [comment]
Thanks to #967 we now have a configurator for 'Tycho'-projects without Eclipse metadat.

And I'm about to create test-cases for them in https://github.com/eclipse-m2e/m2e-core/pull/979.
I think your use-case is covered, but if not please leave a comment about missing parts.
